Microsoft.Toolkit.Uwp.UI.Controls/TextBoxMask/TextBoxMask.Properties.cs

```diff
@@ -0,0 +1,104 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// TextBox Mask property allows a user to more easily enter fixed width text in TextBox control
+    /// where you would like them to enter the data in a certain format
+    /// </summary>
+    public partial class TextBoxMask
+    {
+        /// <summary>
```

Comment:
Need more description around the different properties than just "Gets ____" and "Sets ____". How does the Mask work? What is allowed? What is the CustomMask? How does it work? What is required?